setup.py

```diff
@@ -20,6 +20,7 @@
         'luigi==2.8.12',
         'grpcio',
         'protobuf',
-        'fs'
+        'fs',
+        'cidr-trie'
```

Comment:
what is this for? why not use existing cidr tool in mizar? please explain.

Reply:
cidr-trie is a tool that focus on trie. It inputs all the cidrs into a trie, and then form the relationship between cidrs.